windows_containers/windows-containers-release-notes-10-16-x.adoc

```diff
@@ -0,0 +1,101 @@
+
+// Copied from 4.15
+Previously, the kubelet on Windows nodes was unable to authenticate with private Elastic Container Registries (ECR) registries. Because of this error, the kubelet was not able to pull images from these registries. With this fix, the kubelet is able to pull images from these registries as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26602[*OCPBUGS-26602*])
+
+// Copied from 4.15
+Previously, on Azure clusters the WMCO would check if an external Cloud Controller Manager (CCM) was being used on the cluster. If a CCM is being used, the Operator would adjust configuration logic accordingly. Because the status condition that the WMCO used to check for the CCM was removed, the WMCO proceeded as if a CCM was not in use. This fix removes the check. As a result, the WMCO always configures the required logic on Azure clusters. (link:https://issues.redhat.com/browse/OCPBUGS-31626[*OCPBUGS-31626*])
+
+// Copied from 4.15
+Previously, the WMCO was logging error messages when any command being run through an SSH connection to a Windows instance failed. This was incorrect behavior because some commands are expected to fail. For example, when the WMCO reboots a node, the Operator runs PowerShell commands on the instance until they fail, meaning the SSH connection rebooted as expected. With this fix, only actual errors are now logged. (link:https://issues.redhat.com/browse/OCPBUGS-20255[*OCPBUGS-20255*])
+
+Previously, after rotating the `kube-apiserver-to-kubelet-client-ca` certificate, the contents of the `kubetl-ca.crt` file on Windows nodes was not being populated correctly. With this fix, after certificate rotation, the `kubetl-ca.crt` file contains the proper certificates. (link:https://issues.redhat.com/browse/OCPBUGS-22237[*OCPBUGS-22237*])
+
+Previously, because of a missing DNS suffix in the kubelet host name on instances that are part of a Windows AD domain controller, the cloud provider failed to find VMs by name. With this fix, the DNS suffix is now included in the host name resolution. As a result, the WMCO is able to configure and join Windows instances that are part of AD domain controller. (link:https://issues.redhat.com/browse/OCPBUGS-34758[*OCPBUGS-34758*])
+
+Previously, registry certificates provided to the cluster by the user were not being loaded into the Windows trust store on each node. As a consequence, image pulls from a mirror registry were failing, because a self-signed CA is required. With this fix, registry certificates are loaded into the Windows trust store on each node. As a result, images can be pulled from mirror registries with self-signed CAs. (link:https://issues.redhat.com/browse/OCPBUGS-36408[*OCPBUGS-36408*])
```

Comment:
```suggestion
Previously, registry certificates provided to the cluster by a user were not loaded into the Windows trust store on each node. As a consequence, image pulls from a mirror registry failed, because a self-signed CA is required. With this fix, registry certificates are loaded into the Windows trust store on each node. As a result, images can be pulled from mirror registries with self-signed CAs. (link:https://issues.redhat.com/browse/OCPBUGS-36408[*OCPBUGS-36408*])
```
- Change to indefinite article `a user` to indicate a generic rather than a specific user